tests/Unit/ControlSystem/ControlErrors/Test_SizeError.cpp

```diff
@@ -520,3 +520,57 @@ SPECTRE_TEST_CASE("Unit.ControlSystem.SizeError", "[Domain][Unit]") {
         horizon_velocity, target_char_speed, expected_control_error);
   }
 }
+
+SPECTRE_TEST_CASE("Unit.ControlSystem.SizeError.HorizonHigherResolution",
```

Comment:
Can you make this a free function and call it from the existing `SPECTRE_TEST_CASE`? Especially on HPC systems there's a lot of overhead to starting a test